modules/linear_boltzmann_solvers/b_discrete_ordinates_solver/sweep/angle_aggregation/angle_aggregation.h

```diff
@@ -26,38 +26,43 @@ namespace lbs
  * The octant based separation is achieved via the notion of "AngleSetGroup"
  * which will group angle sets for each quadrant or octant
  * (depending on 2D or 3D).*/
+
 class AngleAggregation
 {
-public:
-  std::vector<AngleSetGroup> angle_set_groups;
-  std::map<uint64_t, std::shared_ptr<SweepBndry>> sim_boundaries;
-  size_t number_of_groups = 0;
-  size_t number_of_group_subsets = 0;
-  std::shared_ptr<AngularQuadrature> quadrature = nullptr;
-
 private:
-  bool is_setup = false;
-  std::pair<size_t, size_t> number_angular_unknowns;
-  bool num_ang_unknowns_avail = false;
+  bool is_setup_;
+  size_t number_of_groups_;
+  size_t number_of_group_subsets_;
+  bool num_ang_unknowns_avail_;
+  std::pair<size_t, size_t> number_angular_unknowns_;
+  std::shared_ptr<MeshContinuum> grid_;
+  std::shared_ptr<AngularQuadrature> quadrature_;
+  std::map<uint64_t, std::shared_ptr<SweepBoundary>> sim_boundaries_;
 
 public:
-  std::shared_ptr<MeshContinuum> grid = nullptr;
+  AngleAggregation(const std::map<uint64_t, std::shared_ptr<SweepBoundary>>& sim_boundaries,
+                   size_t number_of_groups,
+                   size_t number_of_group_subsets,
+                   std::shared_ptr<AngularQuadrature>& quadrature,
+                   std::shared_ptr<MeshContinuum>& grid);
 
-  /** Sets up the angle-aggregation object. */
-  AngleAggregation(const std::map<uint64_t, std::shared_ptr<SweepBndry>>& in_sim_boundaries,
-                   size_t in_number_of_groups,
-                   size_t in_number_of_group_subsets,
-                   std::shared_ptr<AngularQuadrature>& in_quadrature,
-                   std::shared_ptr<MeshContinuum>& in_grid);
+  std::vector<AngleSetGroup> angle_set_groups_;
```

Comment:
I think we use trailing `_` for protected and private member variables. The public ones do not have it. See [coding standard](https://open-sn.github.io/opensn/coding_standard.html#classes-and-structs).

Reply:
Oh, good catch! I originally made that private and didn't fix it when I made it public.

Reply:
Fixed.